packages/pulp/python-aiofiles/python-aiofiles.spec

```diff
@@ -0,0 +1,54 @@
+# Created by pyp2rpm-3.3.3
+%global pypi_name aiofiles
+
+Name:           python-%{pypi_name}
+Version:        0.4.0
+Release:        1%{?dist}
+Summary:        File support for asyncio
+
+License:        Apache 2.0
+URL:            https://github.com/Tinche/aiofiles
+Source0:        https://files.pythonhosted.org/packages/source/a/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
```

Comment:
Can we use the `%pypi_source` macro or does that break other tools?

Reply:
I intentionally didn't use it, as it breaks with packages where the package name does not match the PyPI name

Reply:
For Django, `pyp2rpm` sets `srcname` to `django` and `pypi_name` to `Django`, as `srcname` wins in `pypi_source` and that 404s.

It might be an issue how pyp2rpm uses these two macros, but I didn't want to investigate further.